PlasterTemplate/PlasterTemplates/GitLab/psake.ps1

```diff
@@ -235,13 +278,23 @@ Task Deploy -Depends TestAfterBuild {
         Write-Error "PowerShell repository API key not found"
     }
 
-    <#
-    "`tTesting for GitLab Personal Access Token"
-    if (!$ENV:GitLab_PAT) {
-        Write-Error "GitLab personal access token not found"
-    }
-    #>
 
+<%
+    if ($PLASTER_PARAM_DeployDocs -eq 'Y') {
```

Comment:
Change as per changes in plaster manifest

Reply:
This has been resolved. Github appears to not be picking it up.